assets/_common/styles/custom-styles.scss

```diff
@@ -146,3 +148,49 @@ figcaption details {
 h2.heading-md {
   margin-top: units(2);
 }
+
+/* checklists and key questions */
+.guide-checklist,
+.key-questions {
+  @include u-font('sans','md');
+  color: color('white');
+  padding: units(1);
+}
+
+.guide-checklist {
+  background-color: color('primary');
+}
+
+.key-questions {
+  background-color: color('primary-darker');
+}
+
+.guide-checklist + ul,
+.key-questions + ul {
+  background-color: color('blue-5');
+  margin-top:0;
+  padding-top: units(1);
+  padding-right: units(2);
+  padding-bottom: units(1);
+}
+
+.guide-checklist + ul li,
+.key-questions + ul li {
+  margin-bottom: units(1);
+}
+
+.guide-checklist + ul {
+  list-style: none;
```

Comment:
[issue] This is working for first level `<ul>`s but not for nested ones, unfortunately.  I'm guessing there's a more elegant fix but this seems to work?

`.guide-checklist + ul, .guide-checklist + ul ul {`

![image](https://github.com/18F/guides/assets/2480492/987d0b08-dac5-4509-b6ec-429d817de0eb)

Reply:
Thanks for catching that—I've added your fix to handle two levels of `ul`, which seems sufficient.